.gitignore

```diff
@@ -13,3 +13,6 @@
 
 # vim .swp files
 *.swp
+
+#Vendor folder
+/vendor
```

Comment:
We don't want to ignore the vendor directory right? If we did our use of deps will be broken.

Reply:
but "For example" when i have Go mod enable and do "go mod vendor" , it will update all of them , and  then git is going to add all of the vendors into the commit , 

unless i dont do go mod vendor

Reply:
eh.. I guess maybe I should switch to `go mod`